whitelist_addresses.example.csv

```diff
@@ -0,0 +1,4 @@
+stars...
+cosmos...
+osmo...
+regen...
```

Comment:
```suggestion
stars1...
cosmos1...
osmo1...
regen1...
```

I think the ones are great because they are separators everyone finds in their address, They also help any sort of prefix conflict between star1 addresses ans stars1 addresses.